StortSpelProjekt/Server/src/main.cpp

```diff
@@ -0,0 +1,25 @@
+#include "Engine.h"
+#include <iostream>
+
+int main() {
+
+	Network network;
+
+	std::string ip;
+
+	std::cin >> ip;
+
+	if (ip == "1") {
+		std::cin >> ip;
+		std::cout << "Connecting to " + ip << std::endl;
+		network.ConnectToIP(ip, 55555);
+	}
+	else {
+		std::cout << "Listening for connections" << std::endl;
+		network.ListenConnection(55555);
+	}

```

Comment:
if-statement structure has to be the following:  
if (true)
{
	//do something
}

Changes these statements to reflect this.